open-metadata-resources/open-metadata-deployment/pom.xml

```diff
@@ -40,4 +40,21 @@
             </modules>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <configuration>
```

Comment:
RAT configuration should be in the top level pom file only.

Reply:
Hi Mandy, I add this exclusion due to the failed rat check when I was building the project locally. This is a workaround. Shall I add this exclusion in the top level pom file or just delete it? Thanks!

Reply:
What was the file type that was failing?

Reply:
several CSV and ISX files

Reply:
Why are these files in the build tree?

Reply:
Hi Mandy, I've raised an issue for this building failure and enclosed the rat.txt.